apps/prairielearn/src/pages/home/home.sql

```diff
@@ -73,7 +73,7 @@ WITH
           'id',
           ci.id,
           'expired',
-          coalesce(d.expired, TRUE)
+          coalesce(d.expired, FALSE)
```

Comment:
A comment here would be nice, since it's not clear why FALSE is being used just from context.

```suggestion
          -- If no access rules exist, it is typically either a sandbox or a
          -- future CI that has not yet been configured. In both cases it should
          -- not be considered expired.
          coalesce(d.expired, FALSE)
```